src/main/java/com/ookawara/AnimeAPI/service/AnimeServiceImpl.java

```diff
@@ -37,4 +37,14 @@ public Anime createAnimeData(String name, int episode){
         animeMapper.createAnimeData(anime);
         return anime;
     }
+
+    @Override
+    public void updateAnimeData(int id,String name,int episode){
+        animeMapper.updateAnimeData(id,name,episode);
```

Comment:
formatterはいれておいてほしい。
```suggestion
    public void updateAnimeData(int id, String name, int episode){
        animeMapper.updateAnimeData(id, name, episode);
```

Reply:
修正しました。
f5e9885917650fe94122fdf5a2fe1ad32b8b1dd0

Reply:
対応ありがとうございます！